config/locales/en.yml

```diff
@@ -1151,9 +1156,20 @@ en:
         hints:
           - For example, shares in a public limited company, unit trusts or capital bonds.
           - Enter 0 if this does not apply.
+        with_partner_hints:
+          - For example, shares in a public limited company, unit trusts or capital bonds. Include any jointly owned by the client and partner.
+          - Enter 0 if this does not apply.
         input: Total of all investments
       valuables:
         label: Valuable items worth £500 or more
+        certificated_with_partner_hints:
+          - If you have more than one valuation for any of these valuable items, use the highest. Keep a record of where the valuation is from for evidence, as it may be checked if you make an application for legal aid.
+          - Include any jointly owned by the client and partner.
+          - Enter 0 if this does not apply.
+        controlled_with_partner_hints:
+          - If you have more than one valuation for any of these valuable items, use the highest. Keep a record of where the valuation is from for evidence, as it may be checked at file audit or assessment stage.
+          - Include any jointly owned by the client and partner.
```

Comment:
`Include any jointly owned by the client and partner.` this should be with client between 1177-1178